test/pragmaUtil/index.ts

```diff
@@ -13,33 +13,12 @@ describe("Process before upload", function() {
     );
   });
 
-  it("should remove @sync-ignore and @sync ignore lines", () => {
```

Comment:
may i know why these tests were removed.
also can you add test for multi settings

Reply:
It was removed since the new approach doesn't need separated methods to accomplish the same and some methods were removed. 
I added a new test for multi-line settings but I can't  run the test because TSLint throws errors in "commons.ts" file. Check the latest commits.